common.mk

```diff
@@ -56,11 +56,11 @@ setup-tests: install-uv install-build-deps ##- Set up a testing environment with
 
 .PHONY: setup-lint
 setup-lint: install-uv install-shellcheck install-pyright install-lint-build-deps  ##- Set up a linting-only environment
-	uv sync --no-install-workspace --extra lint --extra types
+	uv sync --no-install-workspace --group lint --group types
 
 .PHONY: setup-docs
 setup-docs: install-uv  ##- Set up a documentation-only environment
-	uv sync --no-dev --no-install-workspace --extra docs
```

Comment:
Why is `--no-install-workspace` not needed anymore?

Reply:
`--no-install-workspace` breaks with some of our apps since we're importing code from the apps to generate command info.